+# zrj: add missing link against -lm in libHSmwc-random-*.so
+# NOTYPE  GLOBAL DEFAULT  UND {exp, log, pow, sqrt}
+# fixes www/hs-wai-app-static (binutils 2.27 ld.gold is unhappy)
--- mwc-random.cabal.orig	2015-03-30 00:25:02.000000000 +0300
+++ mwc-random.cabal
@@ -36,6 +36,7 @@ extra-source-files:
   test/visual.hs
 
 library
+  extra-libraries: m
   exposed-modules:
     System.Random.MWC
     System.Random.MWC.Distributions
